packages/desktop/views/dashboard/drawers/dapp-config/views/InputConnectionCodeDrawer.svelte

```diff
@@ -44,18 +45,19 @@
     })
 </script>
 
-<DrawerTemplate title={localize('views.dashboard.drawers.dapps.inputConnectionCode.title')} {drawerRouter}>
+<DrawerTemplate title={localize(`${localeKey}.title`)} {drawerRouter}>
     <div class="h-full flex flex-col gap-4 justify-between px-6">
         <div class="flex flex-col gap-4">
-            <Alert variant="info" text={localize('views.dashboard.drawers.dapps.inputConnectionCode.hint')} />
-            <TextInput
-                bind:value={walletConnectUri}
-                {error}
-                label={localize('views.dashboard.drawers.dapps.inputConnectionCode.inputLabel')}
+            <SidebarToast
+                color="neutral"
+                header={localize(`${localeKey}.addDapp`)}
+                body={localize(`${localeKey}.addDappDescription`)}
+                dismissable={false}
             />
+            <TextInput bind:value={walletConnectUri} {error} label={localize(`${localeKey}.inputLabel`)} autofocus />
```

Comment:
The autofocus is problematic here. I get a runtime error.

Steps to reproduce:
1. Open dApp drawer
2. Click connect to a dApp
3. Close drawer
4. Open dApp drawer again